app/frontend/utils/formio-component-traversal.ts

```diff
@@ -47,6 +47,21 @@ export const combineComplianceHints = (
   const blocksList = findPanelComponents(updatedJson.components)
   blocksList.forEach((panelComponent) => {
     if (blocksLookups[panelComponent.id]) {
+      if (blocksLookups[panelComponent.id]?.tip) {
+        panelComponent["tip"] = blocksLookups[panelComponent.id].tip
+      }
+      if (blocksLookups[panelComponent.id]?.["enabledElectiveFieldIds"]) {
+        panelComponent.components.forEach((subComp) => {
+          if (subComp.elective) {
+            subComp["conditional"] = {
+              show: blocksLookups[panelComponent.id]?.enabledElectiveFieldIds?.includes(subComp.id),
+              when: null,
+              eq: "",
+            }
+          }
+        })
+      }
```

Comment:
if a jurisdiction does not make any customization this conditional will not pass `if (blocksLookups[panelComponent.id]?.["enabledElectiveFieldIds"])`. This is means if a field is marked elective, then we will display them by default which we not want. Remove the conditional and just have the following for it to work:

```
panelComponent.components.forEach((subComp) => {
          if (subComp.elective) {
            subComp["conditional"] = {
              show: blocksLookups[panelComponent.id]?.enabledElectiveFieldIds?.includes(subComp.id) ?? false,
              when: null,
              eq: "",
            }
          }
        })
```